platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/SwirldsPlatform.java

```diff
@@ -301,6 +292,8 @@ public class SwirldsPlatform implements Platform {
      * Encapsulated wiring for the platform.
      */
     private final PlatformWiring platformWiring;
+    /** thread-queue responsible for hashing states */
+    private final QueueThread<ReservedSignedState> stateHashSignQueue;
```

Comment:
I'm assuming this is something that we will put into the wiring framework soon?

Reply:
This is not new, it has just moved. This is the last component of the main pipeline we have outstanding